_ingest-pipelines/processors/pipeline.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: default
+title: Pipeline
+parent: Ingest processors
+nav_order: 200
+---
+
+# Pipeline processor
+
+The `pipeline` processor defines a series of processing steps to be executed on documents during the ingestion or update process. This is particularly useful when you need to perform complex data transformations or apply multiple processing steps to your data before it is indexed.
+
+The following is the syntax for the `pipeline` processor:
+
+```json
+{
+  "pipeline": {
+    "name": "<pipeline_name>",
+    "description": "<pipeline_description>",
+    "processors": [
+      {
+        "<processor_type>": {
+          "<processor_config>": "<processor_value>"
+        }
+      },
+      {
+        "<processor_type>": {
+          "<processor_config>": "<processor_value>"
+        }
+      }
+    ]
+  }
+}
+```
+{% include copy-curl.html %}
+
+## Configuration parameters
+
+The following table lists the required and optional parameters for the `pipeline` processor.
+
+Parameter | Required/Optional | Description |
+|-----------|-----------|-----------|
+`name` | Required	| The name of the pipeline to execute.
+`processors` | Required | An array of other processor configurations to execute in the specified order.
```

Comment:
`processors` and `ignore_missing_pipeline ` should be removed, this processor doesn't have these parameters.